internal/data/script_utils.go

```diff
@@ -0,0 +1,38 @@
+package data
+
+import (
+	"encoding/hex"
+	"hash/crc32"
+
+	"github.com/nervina-labs/cota-nft-entries-syncer/internal/biz"
+	"github.com/nervina-labs/cota-nft-entries-syncer/internal/data/blockchain"
+)
+
+func GenerateSenderLock(entry biz.Entry) (lockScript biz.Script, lockHashStr string, lockHashCRC32 uint32, err error) {
```

Comment:
It is recommended that this method only returns `LockScript`, put `lockHash` and `CRC` to other function

Reply:
1. follow the single responsibility principle

Reply:
OK, maybe your idea is better.

Reply:
https://github.com/nervina-labs/cota-nft-entries-syncer/pull/14/commits/199e99a667a50c1859bc11d943d84bef845d7c40 cc @shaojunda